contribute.md

```diff
@@ -0,0 +1,5 @@
+# How to Contribute to the Model Zoo
```

Comment:
Need to add info that any model submitted must pass ONNX model checker. (it's manually run for now until we setup CI)

Reply:
Added model checker info in PR template with code on how to perform the check.. Thanks!

Reply:
@rahulbhalley - Can you sign the CLA?